python/cudf/cudf/_lib/csv.pyx

```diff
@@ -554,7 +554,7 @@ cdef data_type _get_cudf_data_type_from_dtype(object dtype) except *:
     # TODO: Remove this work-around Dictionary types
     # in libcudf are fully mapped to categorical columns:
     # https://github.com/rapidsai/cudf/issues/3960
-    if cudf.api.types._is_categorical_dtype(dtype):
+    if isinstance(dtype, cudf.CategoricalDtype) or dtype == "category":
```

Comment:
```suggestion
    if isinstance(dtype, cudf.CategoricalDtype) or (isinstance(dtype, str) and dtype == "category"):
```

Reply:
Thanks. Finally got around to incorporating this change